### ISSUE TITLE
[2.1 branch] Error adding new image

### ISSUE THREAD

**Issue by spacekitteh** [glutanimate/image-occlusion-enhanced#46]
Once an image is selected in the "choose file" dialog box:

``` 
File "C:/Users/spacekitteh/AppData/Roaming/Anki2\addons21\image_occlusion_enhanced\utils.py", line 26, in path2url
    'file:', urllib.request.pathname2url(path.encode('utf-8')))
  File "nturl2path.py", line 48, in pathname2url
<class 'TypeError'>: a bytes-like object is required, not 'str'
```

This is apparently due to path.encode returning a bytes-like object, not a string, and the error is misleading: https://stackoverflow.com/questions/38081511/need-help-understanding-bytes-like-object-vs-str-in-python-3-5

**Comment by spacekitteh** [comment]
This appears to be a Windows-specific problem.

**Comment by glutanimate** [comment]
Thanks for the report! I'll try to fix this soon.

**Comment by spacekitteh** [comment]
It's times like these that I kinda regret not knowing Python!

**Comment by glutanimate** [comment]
@spacekitteh:

Could you please see if this works now? khonkortisan was kind enough to submit a PR that should fix the encoding error on 2.1.

**Comment by glutanimate** [comment]
Closing this as it appears to be fixed.
